src/Mailgun/Resource/Api/Tag/StatisticsResponse.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+/*
+ * Copyright (C) 2013-2016 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Resource\Api\Tag;
+
+use Mailgun\Resource\ApiResponse;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+final class StatisticsResponse implements ApiResponse
+{
+    /**
+     * @var string
+     */
+    private $tag;
+    /**
+     * @var string
+     */
+    private $description;
+    /**
+     * @var string
+     */
+    private $resolution;
```

Comment:
Do you mind to add an empty line between each property and the next annotation please? Although this is a minor update and has no effect, I believe it helps to improve the readability.